src/app/table/data-table/terra-data-table.component.ts

```diff
@@ -132,14 +132,11 @@ export class TerraDataTableComponent<S extends TerraBaseService, D extends Terra
     {
         this._rowList = value;
         
-        if(this.rowList != null)
-        {
-            this.rowList.forEach(
-                row =>row.contextMenuLinkList.forEach(
-                    l => l.subject.subscribe(
-                        val=> val.clickFunction(val)))
-            );
-        }
+        this.rowList.forEach(
+            row => row.contextMenuLinkList.forEach(
+                l => l.subject.subscribe(
+                    val => val.clickFunction(val)))
+        );
```

Comment:
types für row, l and val ergänze.
renaming für l and val (irgendwas sinnvolleres, was man auch versteht)